swagger.yaml

```diff
@@ -385,6 +385,54 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /v1/users/{userId}/readonly-access:
```

Comment:
I think this endpoint should be mentioned in the config file. It's a draft of the config, but it's better to keep it up to date.